app/src/main/java/org/coepi/android/cen/CenRepo.kt

```diff
@@ -124,11 +128,13 @@ class CenRepo(private val cenApi: CENApi, private val cenDao: RealmCenDao, priva
     }
 
     fun periodicCENKeysCheck() {
+        Log.i(TAG_LOG, "periodicCENKeysCheck!")//OK: I see this in the log!
```

Comment:
Remove unnecessary logs

Reply:
also this may be useful when application crashes at startup